### PR TITLE
Add precisions about annotations autoloading

### PR DESCRIPTION
**Pull request by l-vo**

### CODE REVIEW

**Comment by SenseException** [comment]
Thank you for the additions. You probably chose your base branch from the information about supported versions in the documentation. Please change your base branch to `1.8`, because it is the current supported version.

#311 was created to fix the version info on the homepage.

**Comment by l-vo** [comment]
@SenseException target branch changed ✅

File: docs/en/index.rst [hunk 1]
```diff
@@ -73,6 +79,13 @@ annotations of a class. A common one is
 
 A reader has multiple methods to access the annotations of a class.
 
+Note that ``AnnotationRegistry::registerLoader('class_exists')`` only works
+if you already have an autoloader configured (i.e. composer autoloader).
+Otherwise, :ref:`please take a look to the other annotation autoload mechanisms <annotations>`.
+
+A reader has multiple methods to access the annotations
+of a class.
```
Comment (guilliamxavier) [inline docs/en/index.rst:87]: This sentence is now duplicated above and below the note

Comment (l-vo) [reply docs/en/index.rst:87]: My bad... Thank you :) Fix in #320

**Pull request merged**